TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift

```diff
@@ -57,20 +69,36 @@ class PushFactory {
 }
 
 extension PushFactory: PushFactoryProtocol {
-  func createFactor(withAccessToken accessToken: String, friendlyName: String, serviceSid: String,
-                    identity: String, pushToken: String?, metadata: [String: String]?,
-                    success: @escaping FactorSuccessBlock, failure: @escaping TwilioVerifyErrorBlock) {
+  // swiftlint:disable:next function_parameter_count
+  func createFactor(
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/eb579290ef458a5ac85d4e99b29e765308b08746/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/PushFactory.swift#L73"></span>Function body should span 40 lines or less excluding comments and whitespace: currently spans 44 lines<br />
<code>function_body_length</code> <code>PushFactory.swift:73</code></td>
    </tr>
  </tbody>
</table>